src/data/cloud-computing-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const cloudComputingQuiz = [
+  {
+    Question:
+      "What does the scalability benefit of cloud-native apps enable?",
```

Comment:
The question is a little confusing. Let's simplify it

```suggestion
      "Which of the following is a benefit for developing cloud native apps?",
```